### PR TITLE
Trajectories test feedback

### PR DESCRIPTION
**Pull request by nomadscientist**
These are some suggested changes based on testing this tutorial: https://training.galaxyproject.org/training-material/topics/single-cell/tutorials/scrna-case_trajectories/tutorial.html#conclusion

It's an awesome tutorial, great work!

### CODE REVIEW

File: topics/single-cell/tutorials/scrna-case_trajectories/tutorial.md [hunk 1]
```diff
@@ -23,9 +23,8 @@ requirements:
         - scrna-case_alevin-combine-datasets
         - scrna-case_basic-pipeline
 tags:
-- single-cell
 - 10x
-- transcriptomics
+
 contributions:
```
Comment (hexylena) [inline topics/single-cell/tutorials/scrna-case_trajectories/tutorial.md:28]: could add yourself to `testing`

Comment (nomadscientist) [reply topics/single-cell/tutorials/scrna-case_trajectories/tutorial.md:28]: I figured I was already on authorship so didn't make sense to

Comment (hexylena) [reply topics/single-cell/tutorials/scrna-case_trajectories/tutorial.md:28]: I think authorship + editing are mutually exclusive, one is definitely the larger contribution, but testing feels like a separate axis? At least to my feeling, it says someone sat down and tested this, and reported bugs, as a separate activity to text editing/authorship"

**Review by MarisaJL — DISMISSED** [review]
I'm happy with these changes

**Review by shiltemann — APPROVED** [review]

**Pull request merged**